guillotina/json/deserialize_content.py

```diff
@@ -117,11 +117,14 @@ async def set_schema(
             else:
                 data_value = data[name] if name in data else None
                 found = True if name in data else False
-            if found:
 
-                if not self.check_permission(write_permissions.get(name)):
+            if found or field.missing_value is not None:
+                if found and not self.check_permission(write_permissions.get(name)):
                     raise Unauthorized("Write permission not allowed")
 
+                if not found and field.missing_value is not None:
+                    data_value = deepcopy(field.missing_value)
```

Comment:
This will happen for each patch for each missing value no matter if the field has already value. I would do it later.

Reply:
What do you think of this: https://github.com/plone/guillotina/pull/1108/files/60b4423d47c92d5955ea7f5140213087cef9564f#r586610866